### ISSUE TITLE
Has anyone been involved in govhack before?

### ISSUE THREAD

**Issue by desiguel** [BrisJS/meetups#137]
Am thinking about participating..

**Comment by billymwlei** [comment]
Yes and winner of 2015 Brisbane.
Any information you wanna know ?

**Comment by desiguel** [comment]
Oh.. heaps!

Worth participating?
Prep allowed? If so, won't it just get filled with ppl who have worked on their project for months before?
Best parts, worst parts?
Best for teams or individuals?
Tips on how to do well (since you did!)?

**Comment by billymwlei** [comment]
Yes, I meet nice ppl. But I think startup weekend is more worth to go (Same month last year)
Not really. 
Best :meet new ppl.Worst: Seems a lot are hobby/semi-pro coder/student 
There will be some time for you to meet new teammate , but lots go there as a group (one from Halfbrick Studios last year) 
Great idea plus good presentation
Hope this can help.

**Comment by iamkevinv** [comment]
@AnnaGerber has been helping at GovHack before, so i'd consider her an expert, if that helps.

**Comment by AnnaGerber** [comment]
I'm one of the organisers again this year. We have a maker node at HSBNE (for making physical entries e.g. think 3D printed data visualisations or Physical computing type entries) as well as the main GovHack event at UQ this year.

Entries must be created completely within the timeframe of the weekend. The only prep you are encouraged to do advance is to take a look at the datasets available and maybe do some data preparation (e.g. it's ok to download the data sets & clean them up a bit if they need it). The prize categories and themes are announced on the Friday evening, so most people start working on their entries then.

The first time I went along to GovHack (in Canberra) I went by myself, met another person who was also without a team, and we ended up teaming up to win one of the national prizes. It's heaps of fun, I learnt some new skills and it was great way to meet interesting people.
